docs/sql-ref-ansi-compliance.md

```diff
@@ -67,10 +67,8 @@ The following subsections present behaviour changes in arithmetic operations, ty
 
 ### Arithmetic Operations
 
-In Spark SQL, arithmetic operations performed on numeric types (with the exception of decimal) are not checked for overflows by default.
-This means that in case an operation causes overflows, the result is the same with the corresponding operation in a Java/Scala program (e.g., if the sum of 2 integers is higher than the maximum value representable, the result is a negative number).
-On the other hand, Spark SQL returns null for decimal overflows.
-When `spark.sql.ansi.enabled` is set to `true` and an overflow occurs in numeric and interval arithmetic operations, it throws an arithmetic exception at runtime.
+In Spark SQL, by default, Spark throws an artithmetic exception at runtime for non-decimal numeric overflows and returns null for decimal type overflows.
```

Comment:
For ANSI mode, Spark throws exceptions for all numeric overflows?

Reply:
? The new sentence says it is for only `for non-decimal numeric overflows` which is consistent with the old docs. You mean it, right , @yaooqinn .

Reply:
The behavior might be described as
```
In Spark SQL, by default, Spark throws an arithmetic exception at runtime for both interval and numeric type overflows.

If `spark.sql.ansi.enabled` is `false`, then the decimal type will produce `null` values and other numeric types will behave in the same way as the corresponding operation in a Java/Scala program (e.g., if the sum of 2 integers is higher than the maximum value representable, the result is a negative number) which is the behavior of Spark 3 or older.
```

Reply:
I agree that it looks much clear and has the missed `interval` part. Thanks!